app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -438,6 +438,7 @@ def center_toolbar_filename_classic
       if %w[auth_key_pair_cloud
             availability_zone
             host_aggregate
+            chargeback_rate
```

Comment:
I'm wondering if we could use the `GenericToolbarMixin#toolbar` method instead of adding anything here. Not sure about it, just an idea.

Reply:
@skateman to be able to use this approach i will need to add `show_list` and `show` methods in the controller and set `@center_toolbar` in those, do you want to go with that approach?

Reply:
Yes please!